src/sentry/users/services/user/model.py

```diff
@@ -55,14 +57,16 @@ class RpcUserProfile(RpcModel):
     is_sentry_app: bool = False
     password_usable: bool = False
     is_password_expired: bool = False
-    session_nonce: str | None = None
+    session_nonce: str | None = Field(repr=False, default=None)
 
 
 class RpcUser(RpcUserProfile):
     roles: frozenset[str] = frozenset()
     permissions: frozenset[str] = frozenset()
     avatar: RpcAvatar | None = None
-    emails: frozenset[str] = frozenset()
+    emails: frozenset[str] = (
+        frozenset()
+    )  # TODO(Gabe): Verify whether we want this to be in the repr
```

Comment:
If we don't keep this, we probably should also hide `useremails` as it has similar content.